src/main/java/org/sonarlint/intellij/trigger/SonarLintSubmitter.java

```diff
@@ -43,16 +49,21 @@
   private final SonarLintJobManager sonarLintJobManager;
   private final SonarLintGlobalSettings globalSettings;
   private final SonarLintAppUtils utils;
+  private final LocalFileExclusions localFileExclusions;
+  private final ProjectBindingManager projectBindingManager;
 
   public SonarLintSubmitter(Project project, SonarLintConsole console, FileEditorManager editorManager, SonarLintTelemetry telemetry,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00107)